learning_paths/api/v1/urls.py

```diff
@@ -25,4 +28,11 @@
         LearningPathUserGradeView.as_view(),
         name="learning-path-grade",
     ),
+    path(
+        "learning-paths/<uuid:learning_path_id>/enrollments/",
+        LearningPathEnrollmentView.as_view(),
+        name="learning-path-enrollments",
+    ),
+    path("enrollments/", FetchEnrollmentsView.as_view(), name="fetch-enrollments"),
```

Comment:
As you have noted in the ticket, we should update all URLs to be prefixed with `learning-paths` for consistency.

Reply:
Update the URLs and verified that they conform to existing conventions as well.

![image](https://github.com/user-attachments/assets/314e5236-fa3a-4a35-83f8-0baddb154639)